Remotion/Core/Core/Collections/DictionaryExtensions.cs

```diff
@@ -27,33 +28,38 @@ namespace Remotion.Collections
   /// </summary>
   public static class DictionaryExtensions
   {
+    [return: MaybeNull]
     public static TValue GetValueOrDefault<TKey, TValue> (this IDictionary<TKey, TValue> dictionary, TKey key)

```

Comment:
I'm not sure when the docs where fixed but IDictionary<TKey,TValue> no longer allows the key to be nullable. Please mark TKey as notnull and add a todo to add arugment null checks.